go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/AdhityaRamadhanus/fasthttpcors v0.0.0-20170121111917-d4c07198763a
 	github.com/coreos/etcd v3.3.18+incompatible // indirect
 	github.com/coreos/go-systemd v0.0.0-20191104093116-d3cd4ed1dbcf // indirect
-	github.com/dapr/components-contrib v0.0.0-20200529222633-5c62707aba5f
+	github.com/dapr/components-contrib v0.8.0
```

Comment:
Version for contrib should start at 0.1.0

Reply:
changing...

Reply:
done

Reply:
This is what the versioning discussion in april said as well for components-contrib.